src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -89,19 +89,19 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
       </div>
       <div class="col-md-12 col-sm-12 col-xs-12 detail-assignee-wrap">
           <span
-            *ngIf="!workItem.relationalData.assignees?.length"
+            *ngIf="addNewWI || !workItem.relationalData.assignees?.length"
             (click)="activeSearchAssignee()" 
             class="pull-left fa fa-user-plus user-assign-icon">
           </span>
           <img 
-            *ngIf="workItem.relationalData.assignees?.length"
+            *ngIf="!addNewWI && workItem.relationalData.assignees?.length"
             (click)="activeSearchAssignee()"
             class="pull-left user-assign-avatar"
             src="{{workItem.relationalData.assignees[0]?.attributes.imageURL + '&s=20'}}" 
             onError="this.src='https://avatars0.githubusercontent.com/u/563119?v=3&s=20'" />
           
           
-          <div class="pull-left detail-assignee-name">
+          <div *ngIf="!addNewWI" class="pull-left detail-assignee-name">
```

Comment:
Same logic as I explained up there. I don't think addNewWI flag is needed anywhere.